Sources/Commands/SwiftTool.swift

```diff
@@ -122,13 +122,13 @@ public class SwiftTool<Options: ToolOptions> {
         // Bind the common options.
         binder.bindArray(
             parser.add(
-                option: "-Xcc", kind: [String].self,
+                option: "-Xcc", kind: [String].self, strategy: .one,
                 usage: "Pass flag through to all C compiler invocations"),
             parser.add(
-                option: "-Xswiftc", kind: [String].self,
+                option: "-Xswiftc", kind: [String].self, strategy: .one,
                 usage: "Pass flag through to all Swift compiler invocations"),
             parser.add(
-                option: "-Xlinker", kind: [String].self,
+                option: "-Xlinker", kind: [String].self, strategy: .one,
```

Comment:
I wonder if we should call this label `nargs`, that sounds better for some reason.